docs/diagrams/wallet-lifecycle/wallet-lifecycle-3-responding-to-moving-funds-state.tikz

```diff
@@ -0,0 +1,153 @@
+\node[revert, below right=of rmft_check2]                          (rmft_revert1) {Transaction reverted};
+\node[check, below right=of rmft_check3]                           (rmft_check4)  {Has at least\\ moving\_funds\_timeout\_reset\_delay\\ passed since moving\_funds\_timeout\\ was reset for the wallet?};
+\node[revert, below left=of rmft_check4, xshift=-3cm]                           (rmft_revert2) {Transaction reverted};
+\node[action, below right=of rmft_check4]                          (rmft_action1) {Reset the moving\_funds\_timeout};
+\node[action, below=of rmft_action1]                               (rmft_action2) {Notification sent};
+\node[note1, right=of rmft_start.south east, anchor=south west]    (rmft_note1)   {Can be successfully called for wallets:\\ 
+                                                                                   \textbf{- MOVING\_FUNDS}};
+\node[note2, above=of rmft_note1.north, anchor=south, yshift=-8mm] (rmft_note2)   {Sender incentivized by:\\ \textbf{Reimbursement, if maintainer}};
+
+% Arrows
+\draw[->] (rmft_start)   -- (rmft_check1);
+\draw[->] (rmft_check1)  -| (rmft_revert2) \no;
+\draw[->] (rmft_check1)  -| (rmft_check2)  \yes;
+\draw[->] (rmft_check2)  -| (rmft_check3)  \no;
+\draw[->] (rmft_check2)  -| (rmft_revert1) \yes;
+\draw[->] (rmft_check3)  -| (rmft_revert2) \no;
+\draw[->] (rmft_check3)  -| (rmft_check4)  \yes;
+\draw[->] (rmft_check4)  -| (rmft_revert2) \no;
+\draw[->] (rmft_check4)  -| (rmft_action1) \yes;
+\draw[->] (rmft_action1) -- (rmft_action2);
+
+% MOVING FUNDS COMMITMENT
+% Nodes
+\node[start, right=of rmft_start, xshift=12cm]                    (mfc_start)   {Somebody sends \textbf{moving funds commitment}};
+\node[check, below=of mfc_start]                                  (mfc_check1)  {Is the\\ source wallet in\\ MOVING\_FUNDS\\ state?};
+\node[check, below right=of mfc_check1]                           (mfc_check2)  {Is sender a\\ member of the\\ source wallet?};
+\node[check, below right=of mfc_check2]                           (mfc_check3)  {Hasn't the\\ target\_wallet\_commitment\\ with the same source\\ wallet been already\\ posted?};
+\node[check, below right=of mfc_check3]                           (mfc_check4)  {Is \# of pending\\ redemptions on\\ the source wallet\\ equal 0?};
+\node[check, below right=of mfc_check4]                           (mfc_check5)  {Is the source\\ wallet free of pending\\ moved funds\\ sweep requests?};
+\node[check, below right=of mfc_check5]                           (mfc_check6)  {Are the\\ target wallets\\ in LIVE state?};
+\node[check, below right=of mfc_check6]                           (mfc_check7)  {Are the target\\ wallets different\\ than source\\ wallet?};
+\node[check, below right=of mfc_check7]                           (mfc_check8)  {Is \# of target\\ wallets equal to\\ min(\#\_of\_live\_wallets,\\ balance/wallet\_transfer\_max)\\ and \>0?};
```

Comment:
I think this condition is too complicated and I think it's incorrect. The # of target wallets must be equal to min(...) and non-zero.

I suggest we replace it with:

> Is the # of the target wallets in the commitment as expected?

EDIT: I see that in the text we have `> 0` but that > is not rendered on the image. Anyway, I would simplify it.